src/definitions/chat.json

```
@@ -2,95 +2,116 @@
 	"commandName": "chat",
 	"commandDescription": {
 		"en-US": "Sends this content with these attachments or edits this message with them in this channel",
-		"fr": "Envoie ce contenu avec ces pièces jointes ou modifie ce message avec ceux-là dans ce salon"
+		"fr": "Envoie ce contenu avec ces pièces jointes ou modifie ce message avec ceux-là dans ce salon",
+		"pt-BR": "Envia o conteúdo com esses anexos ou edita a mensagem com eles nesse canal"
 	},
 	"postSubCommandName": "post",
 	"postSubCommandDescription": {
 		"en-US": "Sends this content in this channel",
-		"fr": "Envoie ce contenu dans ce salon"
+		"fr": "Envoie ce contenu dans ce salon",
+		"pt-BR": "Envia esse conteúdo pra esse canal"
 	},
 	"patchSubCommandName": "patch",
 	"patchSubCommandDescription": {
 		"en-US": "Edits this message with this content in this channel",
-		"fr": "Modifie de message avec ce contenu dans ce salon"
+		"fr": "Modifie de message avec ce contenu dans ce salon",
+		"pt-BR": "Edita a mensagem com esse conteúdo nesse canal"
 	},
 	"attachSubCommandName": "attach",
 	"attachSubCommandDescription": {
 		"en-US": "Adds at this position this attachment to this message in this channel",
-		"fr": "Ajoute à cette position cette pièce jointe à ce message dans ce salon"
+		"fr": "Ajoute à cette position cette pièce jointe à ce message dans ce salon",
+		"pt-BR": "Adiciona esse anexo nessa posição pra essa mensagem nesse canal"
 	},
 	"detachSubCommandName": "detach",
 	"detachSubCommandDescription": {
 		"en-US": "Removes at this position the attachment from this message in this channel",
-		"fr": "Retire à cette position la pièce jointe de ce message dans ce salon"
+		"fr": "Retire à cette position la pièce jointe de ce message dans ce salon",
+		"pt-BR": "Remove o anexo nessa posição dessa mensagem nesse canal"
 	},
 	"channelOptionName": "channel",
 	"channelOptionDescription": {
 		"en-US": "Some channel",
-		"fr": "Un salon"
+		"fr": "Un salon",
+		"pt-BR": "Algum canal"
 	},
 	"messageOptionName": "message",
 	"messageOptionDescription": {
 		"en-US": "Some message",
-		"fr": "Un message"
+		"fr": "Un message",
+		"pt-BR": "Aluma mensagem"
```

I don't speak Portuguese, but that looks like a typo?
```suggestion
		"pt-BR": "Alguma mensagem"
```